wiki/Community/Discord_servers/en.md

```diff
@@ -1,9 +1,82 @@
+---
+tags:
+  - development
+  - osu!dev
+---
+
 # Discord servers
 
-Index page for all articles regarding community [Discord](https://discord.com/) servers.
+As the community grows and expands, several [Discord](https://discord.com/) servers have been made and maintained by the community to specialize in the game's many different facets.
+
+Each server provides a space with a specific purpose in mind, where players get together to discuss and work on various different projects. This index aims to serve as an index where users can find a subcommunity they can best fit in.
+
+## Community
+
+These are servers made by the community, for the community. They do not have a specific purpose. However, they serve as community hubs for users to interact in and be up to date with different aspects of the game.
+
+| Name | Owner | Description |
+| :-- | :-- | --- |
+| [osu!](https://discord.gg/ppy) | [peppy](https://osu.ppy.sh/users/2) | The **osu!** Discord server, previously known as **osu!dev**, is osu!'s official Discord server for development-related purposes. The server functions as a hub for people to discuss and work on osu!'s open source and community projects. This is the main place to contribute to osu!'s development and get in touch with the osu! team.  |
+| Aiess Project | - | - |
+
+### Project Loved
+
+<!-- This section as a whole needs to be discussed. Last time someone suggested listing the Loved suite of servers in the Project Loved wiki article, not everyone was on board. Moreover, some of its members and pepy themselves are not really happy on how everything is split apart. Question is whether we want to promote these servers while they exist, or ignore them in favour of the #osu-loved channel in the dev server -->
+
+[Project Loved](/wiki/Community/Project_Loved/) organizes its workflows in different servers, divided by the ruleset which they specialize in. Moreover, the official osu! Discord server provides the `#osu-loved` for the community to discuss about the project as a whole.
+
+| Name | Owner | Description |
+| :-- | :-- | --- |
+| [osu! Project Loved](https://discord.gg/gn58Uk5sTE) | - | - |
+| [osu!catch Project Loved](https://discord.gg/phgtyS4UCh) | - | - |
+| [osu!mania Loved Community](https://discord.gg/Ededv7m) | - | - |
+| [Project Loved: Taiko](https://discord.gg/GhfjtZ6) | - | - |
+
+## Gameplay
+
+These servers are focused on giving users a place to discuss osu!'s main attraction — the gameplay! They serve as hubs for players to interact with their daily gaming ventures in osu!.
+
+| Name | Owner | Description |
+| :-- | :-- | --- |
+| [osu! University](https://discord.gg/QubdHdnBVg ) | - | - |
+| [osu! Medal Hunters](https://discord.gg/osumedalhunters) | - | - |
```

Comment:
This is NOT the correct link at all, also, please use this if we are getting rid of `vanity url + alternative link`
```suggestion
| [osu! Medal Hunters](https://discord.gg/8qpNTs6) | - | - |
```

Reply:
These were all the links present in https://github.com/ppy/osu-wiki/issues/6904. Suffice to say, it needs to be double checked.

Reply:
> These were all the links present in #6904. Suffice to say, it needs to be double checked.

I'd rather check actual existing articles instead of it tbh, because they are the ones with up-to-date info, but maybe that's just me

Reply:
> This is a WIP proposal. It is not even close to ready to be merged.

Refer to this. This needs to go through the different server's owners at some point so they have an agency on the description. That will be done once the core idea of the proposal is accepted.

Link has been changed.